src/Sylius/OrderProcessing/OrderFeeProcessor.php

```diff
@@ -263,4 +236,48 @@ private function getDelivery(OrderInterface $order)
 
         return $delivery;
     }
+
+    private function lazyUpsertAdjustment(
```

Comment:
why is it `lazy` ?

Reply:
Indeed, now it's not lazy. The first version was. I'll rename it